src/core/agent/services/signifyNotificationService.ts

```diff
@@ -448,6 +485,10 @@ class SignifyNotificationService extends AgentService {
         metadata.multisigId = multisigNotification[0].exn?.a?.gid;
       }
     }
+    if (event.a.r === NotificationRoute.ExnIpexGrant) {
+      metadata.linkedGroupRequests = {};
+    }
```

Comment:
Do we need this if statement? It should be `undefined` by default right - won't the rest of the code work with `undefined` when receiving the first admit?

Reply:
Now we have `metadata.linkedGroupRequests = undefined;` - do we need the line at all? just to clean it up